gradlew

```diff
@@ -28,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS='"-Xmx64m"'
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
Was this an accident?
```suggestion
DEFAULT_JVM_OPTS='"-Xmx64m"
```

Reply:
That's the defaults coming out of Gradle. Running `./gradlew wrapper` will update your `gradle/` directory as well as your `gradlew` wrappers with what the selected Gradle version ships.

Reply:
Hmm that's interesting. Alright.